src/sentry/static/sentry/app/components/forms/multipleCheckboxField.jsx

```diff
@@ -43,35 +45,55 @@ export default class MultipleCheckboxField extends FormField {
   };
 
   render() {
+    let {
+      required,
+      className,
+      disabled,
+      disabledReason,
+      label,
+      help,
+      choices,
+      style
+    } = this.props;
     let error = this.getError();
-    let className = 'control-group';
-    if (error) {
-      className += ' has-error';
-    }
+    let cx = classNames(className, 'control-group', {
+      'has-error': error
+    });
+    // Hacky, but this isn't really a form label vs the checkbox labels, but
+    // we want to treat it as one (i.e. for "required" indicator)
+    let labelCx = classNames({
+      required
+    });
+    let shouldShowDisabled = disabled && disabledReason;
+
     return (
-      <div className={className}>
-        <label className="control-label">
-          {this.props.label}
-          {this.props.disabled &&
-            this.props.disabledReason &&
-            <span className="disabled-indicator tip" title={this.props.disabledReason}>
-              <span className="icon-question" />
-            </span>}
-        </label>
-        {this.props.help && <p className="help-block">{this.props.help}</p>}
-        {error && <p className="error">{error}</p>}
-        <div className="controls control-list">
-          {this.props.choices.map(([value, label]) => {
+      <div style={style} className={cx}>
```

Comment:
why cx? is this a shorthand for classname? hadn't seen it

Reply:
just a short var name for class name, I saw it used when the `classnames` lib first came out and it just stuck w/ me. helps when you destructure `className` from `props`.

Reply:
hm, i like it